gemini/prompts/prompt_optimizer/vertex_ai_prompt_optimizer_ui.ipynb

```diff
@@ -0,0 +1,907 @@
+{
```

Comment:
<p>After step 6, we need a Clear section to delete all created resources. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/1099/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>I saw that in the template, but since this is intended as a UI rather than just a notebook showcasing the product, I think the users should keep the artifacts rather than deleting them.</p>